core/jvm/src/main/scala/zio/metrics/jvm/DefaultJvmMetrics.scala

```diff
@@ -0,0 +1,37 @@
+package zio.metrics.jvm
+
+import zio._
+
+trait DefaultJvmMetrics
+
+/** JVM metrics, compatible with the prometheus-hotspot library */
+object DefaultJvmMetrics {
+  val collectDefaultJvmMetrics: ZManaged[Has[Clock] with Has[System], Throwable, Unit] =
```

Comment:
This is going to be faster if you use `foreachParDiscard`. Might also change if you end up composing at the `App` level instead of the `ZManaged` level.